integrations/kafka_connector/src/main/java/software/amazon/timestream/utility/TimestreamWriter.java

```diff
@@ -81,6 +139,8 @@ public TimestreamWriter(final DataModel schemaDefinition, final TimestreamSinkCo
      * @param sinkRecords List of incoming records from the source Kafka topic
      */
     public List<RejectedRecord> writeRecords(final AWSServiceClientFactory clientFactory, final Collection<SinkRecord> sinkRecords) {
+        Boolean writeToLiveAnalytics = true;
```

Comment:
Are these configurable?

Reply:
Good catch, this was old code, replaced with configurable settings that were already implemented, also in latest commit